lib/mixlib/install/product.rb

```diff
@@ -193,9 +193,33 @@ def lookup(key, version = :latest)
 
   product "automate" do
     product_name "Chef Automate"
-    package_name "delivery"
-    ctl_command "delivery-ctl"
-    config_file "/etc/delivery/delivery.rb"
+    package_name do |v|
+    #TODO: Change the version to the promoted release version of automate package
+      v < version_for("8.0.0") ? "delivery" : "automate"
```

Comment:
these versions will be determined once we have an actual release of automate. (the assumption here is that, thatversion will be greator that any of the delivery versions relesed so far)
for now: 8.0.0 is the first version of automate to be released
4.4.4 is the version of delivery where automate-ctl was enabled